website/docs/installation/documentdb/rpm.md

```diff
@@ -0,0 +1,93 @@
+---
+sidebar_position: 4
+---
+
+# RPM package
+
+To install the `.rpm` packages for DocumentDB, you can use the `rpm` tool.
+
+We provide different `.rpm` packages for various deployments on [our release page](https://github.com/FerretDB/documentdb/releases/).
+
+- For most use cases, we recommend using the production package (e.g., `documentdb.rpm`).
+- For debugging purposes, use the development package (contains either `-dev` or `-dbgsym` suffix e.g., `documentdb-dev.rpm`/`documentdb-dbgsym.rpm`).
```

Comment:
Isn't `dbgsym` is debian thing? I don't recall such suffix :thinking:  Also is there really development package?

Reply:
Yeah, it is simply incorrect and not actually tested